content/docs/docs-nav.yaml

```diff
@@ -96,6 +96,7 @@ nav:
           - Required metadata: '/docs/details/required-metadata/'
           - Requesting plugins: '/docs/details/requesting-plugins/'
       - Setting secrets: '/docs/details/setting-secrets/'
+      - Role based access control: '/docs/details/rbac/'
```

Comment:
Let's call this permissions or something similar instead?

Reply:
done.